lib/rspec/support/version_checker.rb

```diff
@@ -1,53 +0,0 @@
-module RSpec
-  module Support
-    LibraryVersionTooLowError = Class.new(StandardError)
-
-    # @private
-    class VersionChecker
```

Comment:
The current `rspec-rails` is not using `VersionChecker` and the `rspec-support` dependency is [pinned to major and minor versions](https://github.com/rspec/rspec-rails/blob/078b29fefb129b7942ba512cc0173b0c6d20c582/rspec-rails.gemspec#L35), so I think removing `VersionChecker` does not break anything.